src/test/kotlin/chapter2/exercises/Exercise_2_2.kt

```diff
@@ -34,6 +34,16 @@ class Exercise_2_2 : WordSpec({
         }
         """!verify ordering of a list of correctly ordered Strings
             based on an ordering HOF""" {
+        """!detect ordering of an empty list as sorted""" {
```

Comment:
Looks great! Would you be able to add them to the solutions too for the sake of consistency?